packages/headless/src/app/engine.ts

```diff
@@ -149,7 +151,7 @@ export interface ExternalEngineOptions<State extends object> {
 }
 
 function getUpdateAnalyticsConfigurationPayload(
-  options: EngineOptions<ReducersMapObject>,
+  options: any,
```

Comment:
:suspect: can you detail/explain, this seems wrong.

Reply:
Fixed. 
Explanation: I was lazy (forgot to fix after temp change) 😉